client/components/Photos.js

```diff
@@ -1,17 +1,18 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
 import axios from 'axios';
 import PropTypes from 'prop-types';
 
 const Photos = ({ currentTrip, currentUser }) => {
   // TODO: CONNECT PHOTOS TO DB
-  // const [photos, setPhotos] = useState([]);
+  const [photos, setPhotos] = useState([]);
 
   useEffect(() => {
     axios.get(`/photos/${currentTrip.id}`)
-      .then(() => {
-        // setPhotos(data);
+      .then(({ data }) => {
+        setPhotos(data);
+        console.log('photos from server', data);
```

Comment:
console.info

Reply:
This change has already been made in my most recent push